caffe2/CMakeLists.txt

```diff
@@ -453,6 +453,7 @@ if(NOT INTERN_BUILD_MOBILE OR NOT BUILD_CAFFE2_MOBILE)
     list(APPEND TORCH_SRCS
```

Comment:
Note that mobile build has set `INTERN_DISABLE_AUTOGRAD` at the line above.
These files are supposed not needed for inference-only mobile build.

Reply:
Good point, it seems like `Functions.cpp` is treated differently than the other codegened files and is included in the mobile build!
So I'll add this one file as well.